src/pages/Products/ProductPage.js

```diff
@@ -1,110 +1,58 @@
-                    <span className="close-icon"><i className="fas fa-heart"></i></span>
-                    <div className="card-body">
-                    <h3 className="card-title">Summer T-shirt</h3>
-                    <small className="card-subtitle fw-semibold">&#8377; 800</small>
-                    <a href=""><button className="btn btn-primary add-cart">Add to Cart</button></a>
-                    </div>
-                </div>
-
-                <div className="card br-sm">
-                    <img className="card-img br-sm img-responsive" src="https://picsum.photos/250/250" alt=""/>
-                    <span className="close-icon"><i className="fas fa-heart"></i></span>
-                    <div className="card-body">
-                    <h3 className="card-title">Summer T-shirt</h3>
-                    <small className="card-subtitle fw-semibold">&#8377; 800</small>
-                    <a href=""><button className="btn btn-primary add-cart">Add to Cart</button></a>
-                    </div>
-                </div>
-
-                <div className="card br-sm">
-                    <img className="card-img br-sm img-responsive" src="https://picsum.photos/250/250" alt=""/>
-                    <span className="close-icon"><i className="fas fa-heart"></i></span>
-                    <div className="card-body">
-                    <h3 className="card-title">Summer T-shirt</h3>
-                    <small className="card-subtitle fw-semibold">&#8377; 800</small>
-                    <a href=""><button className="btn btn-primary add-cart">Add to Cart</button></a>
-                    </div>
-                </div>
-
-                <div className="card br-sm">
-                    <img className="card-img br-sm img-responsive" src="https://picsum.photos/250/250" alt=""/>
-                    <span className="close-icon"><i className="fas fa-heart"></i></span>
-                    <div className="card-body">
-                    <h3 className="card-title">Summer T-shirt</h3>
-                    <small className="card-subtitle fw-semibold">&#8377; 800</small>
-                    <a href=""><button className="btn btn-primary add-cart">Add to Cart</button></a>
-                    </div>
+                <div className="product-text">
+                    <h1 className="">Products</h1>
                 </div>
 
+                {products.map(
+                    ({
+                        _id,
+                        title,
+                        categoryName,
+                        prodImg,
+                        rating,
+                        reviews,
+                        originalPrice,
+                        discountPrice,
+                        discountRate,
+                    }) => (
+                        <ProductCard
+                            key={_id}
+                            title={title}
```

Comment:
here instead of giving so many props you can just pass product={product} and then destructure and use the elements in product prop in ProductCard component.